xcode/Subconscious/Shared/Components/FirstRun/RecoveryPhraseView.swift

```diff
@@ -43,11 +43,11 @@ struct RecoveryPhraseView: View {
 }
 
 enum RecoveryPhraseAction: Hashable {
-    case setPhrase(_ phrase: String)
+    case setPhrase(_ phrase: RecoveryPhrase?)
 }
 
 struct RecoveryPhraseModel: ModelProtocol {
-    var phrase: String = ""
+    var phrase: RecoveryPhrase?
```

Comment:
Use RecoveryPhrase in model to prevent accidentally leaking if stringifying model.